README.md

```diff
@@ -674,7 +674,7 @@ fireEvent.change(getByLabelText(/picture/i), {
 **Keyboard events**: There are three event types related to keyboard input - `keyPress`, `keyDown`, and `keyUp`. When firing these you need to reference an element in the DOM and the key you want to fire.
 
 ```javascript
-fireEvent.keyDown(domNode, {key: 'Enter', code: 13})
+fireEvent.keyDown(domNode, {key: 'Enter', keyCode: 13})
```

Comment:
Thanks, but `keyCode` is [a deprecated standard](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode).

It should work to pass it if there's code that relies on `keyCode`, but our docs will use the [standard `code` property](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code).

Thanks anyway!

Reply:
Fixed my use case to avoid deprecated standard. The tests work now.
Thank you!

Reply:
Also, (if it's useful) we didn't have this library at the most current version.